components/com_kunena/admin/install/install.xml

```diff
@@ -136,6 +136,9 @@
 		<field name="alias"
 		       type="varchar(255)"
 		       null="0"/>
+		<field name="icon"
```

Comment:
I will take a look on the evening, matias in the past had said to user params row in category table instead to add a new row